src/main/webapp/script.js

```diff
@@ -224,5 +231,9 @@ function changeText() {
 function login(){
   fetch("/login").then(response => response.json()).then((user) => {
     document.getElementById("user").innerText = user;
+    var currentUser = user;
+    var myArray = ["jiaxinz@google.com","gagomez@google.com","hilakey@google.com","sasymer@google.com","williamdc@google.com"];
```

Comment:
This doesn't seem to be used.

Reply:
I noticed this method is only used outside of the prototype and it has been removed.